GeoLib/PointVec.h

```diff
@@ -143,6 +145,10 @@ class PointVec : public TemplateVec<Point>
 	 */
 	std::vector<std::size_t> _pnt_id_map;
 
+	/// The reverse map to the name to id map, for fast lookup of the name to a
+	/// given point id.
+	std::vector<std::string> _id_to_name_map;
```

Comment:
A small util (bc30338) was used to measure the memory consumption of the [master](https://github.com/TomFischer/ogs/tree/Test) and a [map version](https://github.com/TomFischer/ogs/tree/GeoLibSpeedupName2PointAccessMapVersion) of this PR (that uses a `std::vector`). The results are surprising -  at least for me:
![comparison](https://cloud.githubusercontent.com/assets/800065/7837275/6649ce7e-0486-11e5-9655-7c691cf67c3f.png)
The first three test series are generated using executables created by clang (debug mode). The last three test series are generated using gcc executables (debug mode). In each test 100000 points are generated. The number of point names increases from the left to the right from 0 percent (no point has a name) to 100 percent (i.e. all points have names). Surprisingly, the implementation without a data structure to manage the id -> name mapping (current master) has the same memory usage than the the implementation using the `std::vector` (this PR). A second surprise is that the `std::map` version does not perform better than the `std::vector` version - even for low load. I have no explanation for this.